Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1808,7 +1847,8 @@ def print_context_data_stats(context_data: dict, stage: str) -> Set[str]:
 
 def perform_long_running_loop(client: Client, offenses_per_fetch: int, fetch_mode: str,
                               user_query: str, events_columns: str, events_limit: int, ip_enrich: bool,
-                              asset_enrich: bool, incident_type: Optional[str], mirror_direction: Optional[str]):
+                              asset_enrich: bool, incident_type: Optional[str], mirror_direction: Optional[str],
+                              first_fetch: str):
```

Comment:
`first_fetch` isn't optional?

Reply:
It's either default 3 days or what the user inserts.

Reply:
It's ok. It just got my attention since you added a new param without a default value to an existing method.
I expected something like this:
```suggestion
                              first_fetch: str = '3 days'):
```